armTemplates/azuredeploy.json

```diff
@@ -853,6 +854,41 @@
 				"ipTags": []
 			}
 		},
+		{
+			"type": "Microsoft.Network/ApplicationGatewayWebApplicationFirewallPolicies",
+			"apiVersion": "2024-01-01",
+			"name": "[variables('ApplicationGatewayWebApplicationFirewallPolicyName')]",
+			"tags": "[parameters('resourceTags')]",
+			"location": "[parameters('resourceLocation')]",
+			"properties": {
+				"customRules": [],
+				"policySettings": {
+					"requestBodyCheck": true,
+					"maxRequestBodySizeInKb": 128,
```

Comment:
Clarifier si ceci doit être augmenté

Reply:
Selon les statistiques des trois derniers mois, la taille des _post bodies_ se distribuent comme suit:
httpMethod_s : POST
min_receivedBytes_d : 183
avg_receivedBytes_d: 1819.0916179996962
max_receivedBytes_d : 1007576
Where 99.993 % of them are under 55300 bytes.